core/src/main/java/com/linecorp/armeria/common/MediaType.java

```diff
@@ -128,14 +128,14 @@
     private static final Map<MediaType, MediaType> KNOWN_TYPES = Maps.newHashMap();
 
     private static MediaType createConstant(String type, String subtype) {
-        MediaType mediaType =
+        final MediaType mediaType =
```

Comment:
IIRC we did not add `final` rigorously in this class because it's forked and we wanted to apply any changes from the upstream in the future.

Reply:
Yeap, let me revert this. :-)